layout-final/css/style.css

```
@@ -37,12 +39,12 @@ body {
 }
 
 #area-logo,
-#area-menu {
+nav {
   padding: 10px;
 
 }
 
-#main-area {
+#main {

```

Esse seletor não vai funcionar pois você está selecionando pelo id main ao invéz da tag main, e a tag lá em baixo está sem id

```suggestion
main {
```

---

resolvido